src/Config/ObjectModel/CliBoolean.cs

```diff
@@ -1,18 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-namespace Azure.DataApiBuilder.Config.ObjectModel;
-
-/// <summary>
-/// Class to represent boolean values in CLI. This is required over primitive boolean types because of the limitation of the CommandLineParser library
```

Comment:
Please move this summary to the `CliBool.cs` file since looks like they serve the same purpose, but a summary is missing in CliBool.